src/MMT.Core/StaticResources.cs

```diff
@@ -43,6 +43,14 @@ public static string UpdateExe
                 return @"AppData\Local\Microsoft\Teams\Update.exe";
             }
         }
+        
+        public static string LaunchExe
+        {
+            get
+            {
+                return @"AppData\Local\Microsoft\Teams\current\Teams.exe";
```

Comment:
Is this a standard way? Can't it be shortened?

Reply:
@BlazeOfSoul It is ok xD